### ISSUE TITLE
Implement first iteration of Table

### ISSUE THREAD

**Issue by trybern** [FHIDev/Fhi.Designsystem#214]
![Image](https://github.com/user-attachments/assets/d7efe04b-9557-49e5-b545-7e3726e0f536)

ref: #205

### Acceptance criteria

- It should be implemented in accordance [with the designs in Figma](https://www.figma.com/design/VgQG6XeEbFOmHIrxyAOACR/FHI-Designsystem?node-id=9206-2129&p=f&t=VPrbPC4I623qVJyY-11)
- It should be accessable from the NPM package and CDN
- It should have automated unit and accessibility tests with appropriate coverage
- It should have a story with an introduction, usage instructions, implementation examples and action interactivity
- It should conform to the Web standard on inputs/outputs (e.g form interactions)
- It should have appropriate component level design tokens that references semantic, and if appropriate, primitive level tokens.
